thread.txt

```diff
@@ -73,7 +71,7 @@ You can also access the mods folder quickly by clicking a button in the "Advance
 
 [COLOR=rgb(184, 49, 47)][U][SIZE=5][B]Understand what your mods do and where they come from[/B][/SIZE][/U][/COLOR]
 Many people simply do not know what the mods they have installed actually do, and download them just because a YouTuber recommended them.
-That's why we've added a little guide for every single mod we include in our installer to understand if you want the features in the mod or not.
+That's why we've added small guides for each mod we offer in our installer to understand if you want the features in the mod or not.
 Along with that, you can easily access the Github page, website, video, or whatever source the mod is from, by simply pressing the buttons next to it.

```

Comment:
```suggestion
That's why we've added links (like to the mod author's page or to the source code) and guides for every single thing in Skyclient, 
```

Reply:
I WILL LITERALLY SHIT YOURSELF

WHEN YOU INSTALL MINECRAFT ON MACOS AND LINUX, IT DOES NOT INSTALL JAVA. THATS SOME WEIRD CRAZY SHIT MICROSOFT DOES. ON MACOS YOU MUST SEPRATELY INSTAL JAVA

Reply:
no it works out of the box

Reply:
Java is installed by default on macos/linux. Besides the web installer exists.